package.json

```diff
@@ -22,6 +22,8 @@
     "@angular/platform-browser": "^5.0.0",
     "@angular/platform-browser-dynamic": "^5.0.0",
     "@angular/router": "^5.0.0",
+    "angular-in-memory-web-api": "^0.5.2",
```

Comment:
This should likely be a dev dependency. right?

Reply:
Yeah, that's probably the right call. I was using it as a stand-in for backend API endpoint, but having it available in the dev context only seems to be a good thing to be explicit with.